starting-files/ConnectorConfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "ServerConfig":{
+    "URL":"https://192.168.0.124/piwebapi/omf",
+    "WebID":"j0U1IjG6kMOEW7mxyHFDShgUEktU0VSVkVSLVpo",

```

Comment:
Please make webID something obvious that user must replace. 
something like this: 
```
    "WebID":"<PI Web ID >",
```